src/AzureCloudEventHub.Tests/AzureNewEventHubResourceTests.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.EventHubs;
+using Xunit;
+
+namespace Squadron.AzureCloudEventHub.Tests
+{
+    public class AzureNewEventHubResourceTests
```

Comment:
Add test also for existing eventhub

Reply:
A test has been added in class AzureExistingEventHubResourceTests